lab4_task1.py

```diff
@@ -0,0 +1,63 @@
+class Stadium:
+
+    def __init__(self, match="None vs None", totalGoals=0, numOfSpectators=0, name="Name", lightInLx=0):
+        self.match = match
+        self.totalGoals = totalGoals
+        self.__numOfSpectators = numOfSpectators
+        self.__name = name
+        self.__lightInLx = lightInLx
```

Comment:
8.I think you should rename it more understandably

Reply:
renamed